src/reference/aws/create-client-certificate-acm.md

```diff
@@ -195,12 +195,12 @@ aws secretsmanager create-secret \
   --tags '[{"Key":"certificate-authority-arn", "Value":"arn:aws:acm-pca:us-east-1:...:certificate-authority/..."}, {"Key":"certificate-arn", "Value":"arn:aws:acm-pca:us-east-1:...:certificate-authority/.../certificate/..."}]'
 ```
 
-This secret can now be used by the Zilla Plus (Public MSK Proxy) to resolve private keys and their corresponding signed certificates to support TLS client authentication.
+This secret can now be used by the Zilla Plus for Amazon MSK to resolve private keys and their corresponding signed certificates to support TLS client authentication.
 
 ::: info
 Note the ARN of the newly created secret for the client certificate's private key.
 :::
 
 ::: warning
-In the example above, private certificates are valid for `365 days`, so you will need to renew the certificate and update the secret value accordingly before expiration. The latest secret value and corresponding private certificate are obtained automatically upon restarting the MSK Proxy instance.
+In the example above, private certificates are valid for `365 days`, so you will need to renew the certificate and update the secret value accordingly before expiration. The latest secret value and corresponding private certificate are obtained automatically upon restarting the Zilla proxy instance.
```

Comment:
```suggestion
In the example above, private certificates are valid for `365 days`, so you will need to renew the certificate and update the secret value accordingly before expiration. The latest secret value and corresponding private certificate are obtained automatically upon restarting the Zilla Plus instance.
```

Reply:
Appologies for not giving you a heads up. Leonid and I were discussing what vernacular to use and landed on using "Zilla Proxy(ies)" and I came up with a more subtle way to indicate when we are talking about zilla plus: https://vordimous.github.io/zilla-docs/next/concepts/kafka-proxies/secure-public-access.html

Reply:
I left the "Instance" word in the AWS troubleshooting since EC2 Instances are being used and bing more verbose there wasn't a big issue.

Reply:
Ah cool, thanks!